hub/tests/path_fixtures.py

```diff
@@ -100,6 +100,7 @@ def local_path(request):
         return
 
     path = _get_storage_path(request, LOCAL)
+    LocalProvider(path).clear()
```

Comment:
tests will fail if `--keep-storage` is called on the previous test run. this fixes it